controller/volume_rebuilding_controller.go

```diff
@@ -96,6 +120,69 @@ func (vbc *VolumeRebuildingController) enqueueVolumeAfter(obj interface{}, durat
 	vbc.queue.AddAfter(key, duration)
 }
 
+func isSettingOfflineReplicaRebuilding(obj interface{}) bool {
+	setting, ok := obj.(*longhorn.Setting)
+	if !ok {
+		return false
+	}
+	return setting.Name == string(types.SettingNameOfflineReplicaRebuilding)
+}
+
+func (vbc *VolumeRebuildingController) enqueueSetting(obj interface{}) {
+	_, ok := obj.(*longhorn.Setting)
+	if !ok {
+		utilruntime.HandleError(fmt.Errorf("received unexpected obj: %#v", obj))
+		return
+	}
+
+	vs, err := vbc.ds.ListVolumesRO()
+	if err != nil {
+		utilruntime.HandleError(fmt.Errorf("failed to list volumes: %v", err))
+		return
+	}
+
+	for _, v := range vs {
+		if v.Spec.OfflineRebuilding != longhorn.VolumeOfflineRebuildingIgnored {
+			continue
+		}
```

Comment:
Should enqueue `enabled` and `ignored`. Right?

```suggestion
		if v.Spec.OfflineRebuilding == longhorn.VolumeOfflineRebuildingDisabled {
			continue
		}
```